Gemfile.lock

```diff
@@ -334,8 +334,8 @@ GEM
       railties (= 4.2.5.2)
       sprockets-rails
     rails-assets-date.format (1.2.3)
-    rails-assets-holderjs (2.9.1)
-    rails-assets-jquery (2.2.0)
+    rails-assets-holderjs (2.9.3)
```

Comment:
Why did you update holderjs? [2.9.3 is on rubygems.org](https://rubygems.org/gems/holder_rails/versions/2.9.3) but not on [rails-assets.org](https://rails-assets.org/).  May be we should downgrade for now?